internal/proxy/github.go

```diff
@@ -175,7 +177,11 @@ func (p *Proxy) ProcessBuffer() {
 								break
 							}
 
-							if resp.StatusCode != 200 && resp.StatusCode != 404 { //Retry later if the response is not 200
+							if resp.StatusCode == 200 {
+								log.WithField("ns", ns).Infof(fmt.Sprintf("Request for %s to %s forwarded.", ns, req.Host))
```

Comment:
why fmt.Sprintf? wouldn't Infof suffice? Also the ns is already logged as a Field so we don't have to repeat and use %q instead of %s so that we know if Host is "" by any chance.

Also can't we return here?

Reply:
no